doc/article/technical_article.tex

```diff
@@ -330,6 +330,19 @@ \subsection{User Interface}
 
 All time entries will be displayed on the time entry page.
 
+\section{Working with Scrum}
+Scrum describes a framework for agile software development. Agile in this
+context means, that requirements and results can evolve during the development.
+The development team uses Scrum for developing trckr and divides the tasks in
```

Comment:
I'd say "... place the tasks into sprints that are 2 weeks long..." because the tasks are not divided, but fitted into a slot